.github/pull_request_template.md

```diff
@@ -23,3 +23,12 @@ Please update the following to link the associated issue. This is required for s
 -->
 
 Fixes: #issue_number
+
+## Contributor checklist
+Please verify that the PR meets the following requirements, where applicable:
+
+- [ ] An overview of proposed schema changes is included in a linked GitHub issue.
+- [ ] A design document PR is created in the [design-notes repository](https://github.com/radius-project/design-notes/), if new APIs are being introduced.
+- [ ] If applicable, design document has been reviewed and approved by Radius maintainers/approvers.
+- [ ] A PR for the [samples repository](https://github.com/radius-project/samples) is created, if existing samples are affected by the changes in this PR.
+- [ ] A PR for the [documentation repository](https://github.com/radius-project/docs) is created, if the changes in this PR affect the documentation or any user facing updates are made.
```

Comment:
Do we want another line item for the recipes repo?
- [ ] A PR for the [recipes repository](https://github.com/radius-project/recipes) is created, if changes in this PR affect existing recipes or require new recipes to be added.

Reply:
Good catch! I thought I already added that in the list, but looks like I missed it. Will update. Thanks!

Reply:
Updated